drivers/dac/ad3552r/ad3552r.h

```diff
@@ -457,6 +457,8 @@ struct ad3552r_init_param {
 
 uint8_t ad3552r_reg_len(uint8_t addr);
 
+uint8_t get_code_reg_addr(uint8_t ch, uint8_t is_dac, uint8_t is_fast);
```

Comment:
why should this function be public ? can you provide a scenario where it's needed ?

also, if it's public, it should have the ad3552r prefix

Reply:
we can use this function to get address of dac or channel register based on mode so that we can directly write to register address.